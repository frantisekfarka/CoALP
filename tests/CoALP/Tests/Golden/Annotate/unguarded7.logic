q(s(X),Y):-p(X,t(Y)).
p(X,t(Y)):-q(s(X),Y).
