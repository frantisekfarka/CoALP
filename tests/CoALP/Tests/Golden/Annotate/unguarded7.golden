q(s(V_2i), V_3i, transform-func-1(V_6c)) :- p(V_2i, t(V_3i), V_6c).
p(V_4i, t(V_5i), transform-func-2(V_7c)) :- q(s(V_4i), V_5i, V_7c).
